source/tool/tests/software/test_util.py

```diff
@@ -48,12 +48,22 @@ def _chipsec_util(self, arg, helper_class=mock_helper.TestHelper):
         chipsec_util._cs = chipset.cs()
         util = chipsec_util.ChipsecUtil()
         util.VERBOSE = True
+        logger.logger().HAL = True
         util.set_logfile(self.log_file)
         err_code = util.main(["chipsec_utils.py"] + arg.split())
         logger.logger().close()
         self.log = open(self.log_file).read()
         self.assertEqual(err_code, 0)
 
+    def _assertLogValue(self, name, value):
+        """Shortcut to validate the output.
+
+        Assert that at least one line exists wihtin the log which matches the
+        expression: \s*name\s*[:=]\s*value.
```

Comment:
suggest just using a description of the expression here (no point in copying code into the comment)...:

expression of the form: name [:=] value